resources/lang/en/admin/hardware/general.php

```diff
@@ -12,6 +12,7 @@
     'clone'  					=> 'Clone Asset',
     'deployable'  				=> 'Deployable',
     'deleted'  					=> 'This asset has been deleted.',
+    'delete_confirm'            => 'Are you sure you want to delete this asset?',
```

Comment:
I don't think we need this? We already have a translation string for "Are you sure you want to delete <item>".

`{{ trans('general.sure_to_delete') }}`

I'd really ideally prefer that be something that takes a variable for better localization, but it would at least mimic what we already have on the table delete buttons.

Reply:
I can fix that in a bit or this weekend.